cassandra/src/test/scala/org/openzipkin/dependencies/spark/CassandraDependencyStoreSpec.scala

```diff
@@ -0,0 +1,39 @@
+package org.openzipkin.dependencies.spark
+
+import com.twitter.util.Await._
+import com.twitter.zipkin.common.Span
+import com.twitter.zipkin.storage.DependencyStoreSpec
+import com.twitter.zipkin.storage.cassandra.{CassandraDependencyStore, CassandraSpanStore}
+import org.junit.BeforeClass
+import org.openzipkin.dependencies.spark.cassandra.ZipkinDependenciesJob
+
+object CassandraDependencyStoreSpec {
+
+  @BeforeClass def ensureCassandra = CassandraFixture.cassandra
+}
+
+class CassandraDependencyStoreSpec extends DependencyStoreSpec {
+  val spanStore = new CassandraSpanStore {
+    /** Deferred as repository creates network connections */
+    override lazy val repository = CassandraFixture.repository
+  }
+
+  override val store = new CassandraDependencyStore {
+    /** Deferred as repository creates network connections */
+    override lazy val repository = CassandraFixture.repository
+  }
+
+  override def processDependencies(spans: List[Span]) = {
```

Comment:
this method block different from the upstream class of the same name. instead of using the dependencystore api directly, we are testing the job by using it to populate the test data.